src/h3lib/lib/algos.c

```diff
@@ -424,15 +430,15 @@ H3Index h3NeighborRotations(H3Index origin, Direction dir, int* rotations) {
  *
  * @param origin Origin location.
  * @param k k >= 0
- * @param out Array which must be of size maxKringSize(k).
+ * @param out Array which must be of size maxGridDiskSize(k).
  * @return 0 if no pentagon or pentagonal distortion area was encountered.
  */
-int H3_EXPORT(hexRange)(H3Index origin, int k, H3Index* out) {
-    return H3_EXPORT(hexRangeDistances)(origin, k, out, NULL);
+int H3_EXPORT(gridDiskUnsafe)(H3Index origin, int k, H3Index* out) {
+    return H3_EXPORT(gridDiskDistancesUnsafe)(origin, k, out, NULL);
 }
 
 /**
- * hexRange produces indexes within k distance of the origin index.
+ * gridDiskUnsafe produces indexes within k distance of the origin index.
```

Comment:
Nit: Should be `gridDiskDistancesUnsafe`, right?

Reply:
Should it? I though the document just said "gridDiskUnsafe"?

Reply:
Unless I'm reading the diff wrong, `gridDiskUnsafe` is the function above this, and this comment refers to `gridDiskDistancesUnsafe` below